.circleci/config.yml

```diff
@@ -1,10 +1,38 @@
 version: 2
+
+workflows:
+  version: 2
+  workflow:
+    jobs:
+      - test-3.6
+
+setup:
+  - &lint
+    run:
+      name: Run flake8
```

Comment:
The name of this step would be wrong but we have recently been breaking these 3 checks into separate steps so it is clearer what failed. So I would just break black, flake8 and isort out.